pkgs/misc/vscode-extensions/nixpkgs-fmt/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, jq, nixpkgs-fmt, vscode-utils}:
+
+let
+  inherit (vscode-utils) buildVscodeMarketplaceExtension;
+in
+buildVscodeMarketplaceExtension {
+  mktplcRef = {
+    name = "nixpkgs-fmt";
+    publisher = "B4dM4n";
+    version = "0.0.1";
+    sha256 = "bf3da4537e81d7190b722d90c0ba65fd204485f49696d203275afb4a8ac772bf";
+  };
+
+  nativeBuildInputs = [
+    jq
+  ];
```

Comment:
is this just needed at build time? It won't be available when the extension is running

Reply:
Yeah it's only needed to set the extension settings
rust-analyzer does something very similar:
https://github.com/NixOS/nixpkgs/blob/a37689ffd0490e3409948b8fe48f36f5c2e43044/pkgs/misc/vscode-extensions/rust-analyzer/default.nix#L33